### ISSUE TITLE
Pin keeps copying previous value

### ISSUE THREAD

**Issue by ksabhnani** [p0deje/Maccy#496]
Great app!

As the title suggests, I updated the value of one of my pins, say from `foo` to `bar`. When I use the shortcut key for the pin, I'm still getting the old value `foo`. I even changed the shortcut key from `u` to `y` and it is still giving me back `foo` when I press Command+Y. I have quit & reopened Maccy multiple times.

Once again, thanks for great work on a fantastic app.

**Comment by p0deje** [comment]
Yes, you can change the title of pin, but actual value remains the same.
